app/com/arpnetworking/database/h2/triggers/AlertsUpdateEtagTrigger.java

```diff
@@ -26,6 +26,6 @@
      * Public no args constructor.
      */
     public AlertsUpdateEtagTrigger() {
-        super("portal.alerts_etag_seq");
+        super("portal.alerts_etags", 15);
```

Comment:
The column (yes, positional) of the organization field in the table the trigger is on.  That's the best that H2 gives us: an array of new and old values.  I'm not pleased with the mechanism.